CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Change Log
 
+## 0.12.2 - 2023-06-28
+
+## What's Changed
+
+### Added
+* Add HTML preset by @alexweininger in [#835](https://github.com/microsoft/vscode-azurestaticwebapps/pull/835)
+* Add actions submenu to workspace view title by @alexweininger in [#838](https://github.com/microsoft/vscode-azurestaticwebapps/pull/838)
+
+### Fixed
+* Fix detecting linked environments by @alexweininger in [#843](https://github.com/microsoft/vscode-azurestaticwebapps/pull/843)
+* Fix creating SWA in browser CodeSpaces by @alexweininger in [#841](https://github.com/microsoft/vscode-azurestaticwebapps/pull/841)
+
+### Engineering
```

Comment:
I'm okay with keeping this, but also don't know if our customers will care at all. I guess it's kind of for us?

Reply:
It's just for us